iocage/__main__.py

```diff
@@ -30,15 +30,17 @@
 import sys
 
 import click
-# This prevents it from getting in our way.
-from click import core
 
-import iocage.lib.ioc_check as ioc_check
-
-core._verify_python3_env = lambda: None
+click.core._verify_python3_env = lambda: None
 user_locale = os.environ.get("LANG", "en_US.UTF-8")
 locale.setlocale(locale.LC_ALL, user_locale)
 
+# inject ./cli to path
+__dirname = os.path.abspath(os.path.dirname(__file__))
+iocage_cmd_folder = os.path.join(__dirname, "cli")
+iocage_lib_folder = os.path.join(__dirname, "lib")
+sys.path = [iocage_cmd_folder, iocage_lib_folder] + sys.path
+
```

Comment:
I agree, this must be changed! Thank you for mentioning the global effect this has.